README.md

```diff
@@ -201,7 +201,9 @@ The player controls can be customized with the following parameters:
 | `playPause`             |   `Boolean`    | `true`  | Display the play/pause button on the control bar     |
 | `progressBar`           |   `Boolean`    | `true`  | Display the progress bar on the control bar          |
 | `time`                  |   `Boolean`    | `true`  | Display the time information on the control bar      |
+| `seekSeconds`           |   `Number`     |  `5`    | Seconds to seek backward/forward with shortcuts      |
 | `volume`                |   `Boolean`    | `true`  | Display the volume button on the control bar         |
+| `volumePercent`         |   `Number`     |  `0.1`   | Percent to increase/decrease volume with shortcuts   |
```

Comment:
The unit percent in the name can be confusing, perhaps `volumeStep`. What do you think?

For the description, `Customize the volume step using shortcuts (between 0 and 1)`

Reply:
Same here, for the name, it's fine.
For the description, I will say something like: `Set the volume step (between 0 and 1) of the increase and decrease shortcuts`.

Reply:
Good for me